code/modules/mob/living/carbon/human/species/station/diona/diona.dm

```diff
@@ -29,7 +29,7 @@
 	slowdown = 7
 	rarity_value = 4
 	hud_type = /datum/hud_data/diona
-	siemens_coefficient = 0.3
+	siemens_coefficient = 0
```

Comment:
Take into consideration that this also makes them immune to taser and stun beams.

Reply:
wasn't aware of this, will work on changing this

Reply:
Should be fixed now